st2chatops.env

```diff
@@ -55,6 +55,25 @@ export ST2_WEBUI_URL="${ST2_WEBUI_URL:-https://${ST2_HOSTNAME}}"
 # Uncomment the following line to force hubot to exit if disconnected from slack.
 # export HUBOT_SLACK_EXIT_ON_DISCONNECT=1
 
+# Microsoft Teams settings (https://github.com/Microsoft/BotFramework-Hubot)
+# Uses the BotFramework
+#
+# export HUBOT_ADAPTER=botframework
+# export BOTBUILDER_APP_ID=aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee
+# export BOTBUILDER_APP_PASSWORD=...
+# Optional:
+# BotBuilder endpoint - defaults to /api/messages
+# export BOTBUILDER_ENDPOINT=...
```

Comment:
A bit nicer to use existing defaults as values for every new ENV variable we list in `st2chatops.env`, instead of `...` where it's harder to guess what the value could be.

Reply:
:+1: